frontend/src/App.js

```diff
@@ -1,11 +1,49 @@
-import React from 'react';
-
-function App() {
-  return (
-    <div className="App">
-      <h1>Mafia Start</h1>
-    </div>
-  );
+import React from "react";
+import WelcomeMenu from "./WelcomeMenu";
+import WaitingRoom from "./WaitingRoom";
+const PHASE = {
+  START: 0,
+  WAITING_ROOM: 1,
+  IN_GAME: 2
+};
+class App extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      phase: PHASE.START
+    };
+
+    this.componentChoser = this.componentChoser.bind(this);
```

Comment:
old syntax, it's not used anymore

Reply:
also, naming

Reply:
nice bro! this is good ref for this issue:
https://medium.com/@nikolalsvk/loosing-bind-this-in-react-8637ebf372cf

Reply:
changed to screenChoser

Reply:
screenChooser

Reply:
done